projects/samples/contests/robocup/controllers/referee/tests/launch_all_tests.sh

```diff
@@ -44,22 +44,22 @@ do
     folder=$(dirname ${test_file})
     test_log="${folder}/test.log"
     referee_log="${folder}/referee.log"
-    msg_prefix="[$NTH_TEST/${#TEST_FILES[@]}] $folder"
+    msg_prefix="[$NTH_TEST/${#TEST_FILES[@]}] ${folder:2}"
 
     if should_run_tests
     then
-        echo "$msg_prefix ..."
-        ./launch_test.sh ${folder} &> ${test_log}
+        printf '%-60.60srunning...' "$msg_prefix"
+        ./launch_test.sh ${folder} | tee ${test_log} > /dev/null
```

Comment:
Could you please add a comment like "Fixed output buffering on Windows" above this line, so that readers of this code are aware that this is a hack to fix some weird behavior on Windows?